tests/Commands/MakeModelCommandTest.php

```diff
@@ -31,3 +31,23 @@
         ->toBeFile()
         ->toPlaceholdersBeReplaced();
 });
+
+
+test('can make model, factory and migration are created using shortcuts', function () {
+    $this->artisan('beyond:make:model User/User -mf --force');
```

Comment:
Is there a reason you are using the `force` flag?

Reply:
No. It should work without a force flag. Actually, I just followed the existing test by adding a shortcut flag.

Reply:
Ah okay, die not see that :D